atlasdb-client/src/main/java/com/palantir/atlasdb/schema/TargetedSweepSchema.java

```diff
@@ -52,7 +52,7 @@ private static Schema generateSchema() {
             dynamicColumns();
                 columnComponent("timestamp_modulus", ValueType.VAR_LONG);
                 columnComponent("write_index", ValueType.VAR_LONG);
-                value(ValueType.BLOB);
+                value(ValueType.STRING);
```

Comment:
[non-blocking] Consider using `TableReferenceAndCell.class` and having a json-based persister for type safety. Might be pretty nontrivial so wouldn't block on this

Reply:
Makes sense